Dockerfile

```diff
@@ -8,5 +8,20 @@ RUN apt-get install ffmpeg -y
 COPY ./app/requirements.txt ./ 
 RUN pip install --no-cache-dir -r requirements.txt
 
+ENV DATABASE_ENGINE='django.db.backends.sqlite3'
```

Comment:
I think the environment variables should have a prefix, like YTSM_DATABASE_ENGINE, to avoid any conflicts with any other software.

Reply:
The variables are also read when not running in a container, that's what I'm worried about.

Reply:
This should be fixed as of https://github.com/bsquidwrd/ytsm/commit/457c0a2fd4c15de30a5315048d01d383f103973f

Reply:
and https://github.com/bsquidwrd/ytsm/commit/8260ed7eea98ed4e0a155c1b9ef65506d5983294